jsonmerge_utils.py

```diff
@@ -5,17 +5,38 @@
 @author: GRENTOR
 """
 import os
+import argparse
 import logging
 #from tqdm import tqdm
 from merge_files import Merge
+
+parser = argparse.ArgumentParser(description="link-check is a broken link identifier")
+parser.add_argument('-ip',
+                    help="Input prefix",
+                    required=True)
+parser.add_argument('-op',
+                    help="Output prefix",
+                    required=True)
+parser.add_argument('-maxFileSize',

```

Comment:
`dest = "max_file_size"`

also use the type parameter for converting to int so that we wont have to do it afterwords:
`type=int`